run.go

```diff
@@ -227,6 +227,10 @@ var runCommand = &cli.Command{
 			Name:  "label-file",
 			Usage: "Read in a line delimited file of labels",
 		},
+		&cli.StringFlag{
+			Name:  "cidfile",
+			Usage: "Write the container ID to the file",
+		},
```

Comment:
Could you also update README.md to include `cidfile`

Reply:
To which category though?

Reply:
`Metadata flags:` would be fine, I think

Reply:
Added